src/client/visualizations/line-chart/line-chart.tsx

```diff
@@ -300,15 +297,24 @@ export class LineChart extends BaseVisualization<LineChartState> {
     });
   }
 
+  highlightRange(): PlywoodRange {
```

Comment:
This implementation is special for line-chart or more generic?

Reply:
This is line-chart specific. I mean, maybe some chart in the future could use this, but right now, it's line-chart only.

Reply:
ok